Fabulous.XamarinForms/templates/content/blank/.template.config/template.json

```diff
@@ -156,7 +156,7 @@
       "type": "parameter",
       "dataType": "string",
       "replaces": "XamarinFormsSdk",
-      "defaultValue": "4.1.0.555618"
+      "defaultValue": "4.2.0.778463"
```

Comment:
Make sure to target the lowest possible version of Xamarin.Forms 4.2 (4.2.0.709249) ;)